.github/workflows/build.yml

```diff
@@ -0,0 +1,13 @@
+name: Build
+on: [pull_request, workflow_dispatch]
+jobs:
+  cocoapods:
+    name: CocoaPods
+    runs-on: macOS-11
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Use Xcode 13
+        run: sudo xcode-select -switch /Applications/Xcode_13.0.app
+      - name: Run pod lib lint
+        run: pod lib lint
```

Comment:
What version of cocoapods does github actions use? Can't seem to find it straightforwardly via documentation, though there are a lot of actions that will help to install and configure it

Reply:
I would assume it's just the latest version. We could include a Gemfile / bundler to make sure each of our development (and this CI) uses the same version.

Reply:
I'm creating a gemfile to add Fastlane, so i can add this there